JavaScript2/Week5/homework.md

```diff
@@ -6,7 +6,6 @@
 All share a video or a resource (this can be a drawing, article or a pod cast) that was helpful for you the last few weeks with learning JavaScript. Please share this in the channel of your class in Slack. Also write as small note about what the resource i about and why you think it's so helpful (you can share more than one if you like).
```

Comment:
should maybe refer to #teach-yourself channel? Just so people get in the habit of using dedicated channels? Or what do you think?

Reply:
Yup, good idea